src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,454 @@
+media notification icons if you want to control the amount of time skipped.
+
+    let skipTime = 10; // Time to skip in seconds
+
+    navigator.mediaSession.setActionHandler('seekbackward', function() {
+      // User clicked "Seek Backward" media notification icon.
+      audio.currentTime = Math.max(audio.currentTime - skipTime, 0);
+    });
+    
+    navigator.mediaSession.setActionHandler('seekforward', function() {
+      // User clicked "Seek Forward" media notification icon.
+      audio.currentTime = Math.min(audio.currentTime + skipTime, audio.duration);
+    });
+
+#### Play / Pause
+
+The "Play/Pause" icon is always shown in the media notification and the related
+events are handled automatically by the browser. If for some reason the default
+behaviour doesn't work out, you can still handle "Play" and "Pause" media events.
+
+    navigator.mediaSession.setActionHandler('play', function() {
+      // User clicked "Play" media notification icon.
+      // Do something more than just playing current audio...
+    });
+
+    navigator.mediaSession.setActionHandler('pause', function() {
+      // User clicked "Pause" media notification icon.
+      // Do something more than just pausing current audio...
+    });
+
+Note: The browser may consider that the web app is not playing media when files
+are seeking or loading. You can override this behaviour by setting
+`navigator.mediaSession.playbackState` to `"playing"` or `"paused"`.  This
+comes in handy when you want to make sure your web app UI stays in sync with
+the media notification controls.
+
+## Notifications everywhere
+
+The cool thing about the Media Session API is that the notification tray is not
+the only place where media metadata and controls are visible. The media
+notification is synced automagically to any paired wearable device. And it also
+shows up on lock screens.
+
+<div class="clearfix"></div>
+<div class="attempt-left">
+  <figure>
+    <img src="/web/updates/images/2017/02/lock-screen.png" alt="Lock Screen">
+    <figcaption>Lock screen</figcaption>
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="/web/updates/images/2017/02/wear.png" alt="Wear Notification">
+    <figcaption>Wear Notification</figcaption>
+  </figure>
+</div>
+<div class="clearfix"></div>
+
+## Make it play nice offline
+
+I know what you're thinking now... *[Service Worker] to the rescue!*
+
+True but first and foremost, you want to make sure **all items in this
+checklist are checked**:
+
+1. All media and artwork files are served with the appropriate `Cache-Control`
+HTTP header. This will allow the browser to cache and reuse previously fetched
+resources. See the [Caching checklist].
+2. Make sure all media and artwork files are served with the
+`Allow-Control-Allow-Origin: *` HTTP header. This will allow third-party web
+apps to fetch and consume HTTP responses from your web server.
+
+### The Service Worker caching strategy
```

Comment:
The term service worker is lower case unless referring to the 'Service Worker API'. See my rules for media source, above. Please check throughout.